fastchat/serve/gradio_block_arena_named.py

```diff
@@ -301,14 +346,19 @@ def bot_response_multi(
             break
 
 
-def flash_buttons():
+def flash_buttons(show_vote_buttons: bool = True):
```

Comment:
We shouldn't flash vote buttons if the text fails the moderation test. essentially, people shouldn't be able to vote if it fails since there will be no output